android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -541,9 +541,7 @@ private void showNTPImage(NTPImage ntpImage) {
             mSuperReferralLogo.setVisibility(View.VISIBLE);
             mCreditText.setVisibility(View.GONE);
             int floatingButtonIcon =
-                GlobalNightModeStateProviderHolder.getInstance().isInNightMode()
-                ? R.drawable.qrcode_dark
-                : R.drawable.qrcode_light;
+                R.drawable.ic_qr_code;
```

Comment:
you can either change the image tint based on light/dark mode or have different icons. based on the code, it seems to be using light mode icon only